docs/de/reference/markdown-basics.md

```diff
@@ -35,7 +35,7 @@ Today, several implementations of the Markdown syntax coexist. The most notewort
 
 ## Zettlr and Markdown
 
-Zettlr itself implements a mixture of different dialects. The editor itself highlights only GitHub flavoured Markdown (plus some extra-elements, which extends Markdown syntax with Zettelkasten elements. Those are described in the respective [chapter on the Zettelkasten method](../academic/zkn-method.md)). If you export your documents to HTML and don't have Pandoc installed, Zettlr will convert your documents using the _GitHub flavoured Markdown syntax_. If available, Zettlr uses Pandoc for exports, which itself reads your Markdown documents using its _Pandoc Markdown syntax_.
+Zettlr itself implements a mixture of different dialects. The editor itself highlights only GitHub flavoured Markdown (plus some Markdown extensions for Zettelkasten elements and other conveniences. The Zettelkasten elements are described in the respective [chapter on the Zettelkasten method](../academic/zkn-method.md), the others are described below). If you export your documents to HTML and don't have Pandoc installed, Zettlr will convert your documents using the _GitHub flavoured Markdown syntax_. If available, Zettlr uses Pandoc for exports, which itself reads your Markdown documents using its _Pandoc Markdown syntax_.
```

Comment:
Having two sentences in brackets somehow sounds uncommon, don't you think …?

Reply:
I can change it to a hyphen? i.e.

> (plus some Markdown extensions for Zettelkasten elements and other conveniences - Zettelkasten elements are described in the respective chapter on the Zettelkasten method, the others are described below)

Don't really mind!

Reply:
Or simply leave the brackets out, because this would make an easy fix!